core/save-git.c

```diff
@@ -194,11 +194,6 @@ static void save_temperatures(struct membuffer *b, struct divecomputer *dc)
 	put_temperature(b, dc->watertemp, "watertemp ", "°C\n");
 }
 
-static void save_airpressure(struct membuffer *b, struct divecomputer *dc)
-{
-	put_pressure(b, dc->surface_pressure, "surfacepressure ", "bar\n");
-}
-
```

Comment:
so we saved the pressure before, now we save it differently. Can you elaborate why this is necessary?

Reply:
See the discussion here:
https://github.com/Subsurface-divelog/subsurface/pull/2082#discussion_r280376552
https://github.com/Subsurface-divelog/subsurface/pull/2082#discussion_r280669523

The old code saved the value of the *dive computer*. The point here is to save the value entered by the *user*.

Reply:
Of course, the air pressure of the dc should still be saved, so this part of the code shouldn't be removed...

Reply:
Real use case: I dive my Galileo at 1200m altitude. Galileo does not report atm. pressure. I can now type in the altitude, gets converted to mbar and saved as the dive.surface_pressure. On the other hand if I use my Petrel I do not want to be in the situation as to override the surface_pressure reported by the Petrel. The original datum is too valuable. I am considering disabling the write access to the text box and combobox if the dive computer has a surface_pressure.

Reply:
@willemferguson: The problem is that here you remove saving the value reported by the dive computer(s). This code should stay in - you only want to write the user supplied value *in addition*.

Reply:
Ha, I understand